bazel/repositories.bzl

```diff
@@ -484,7 +484,20 @@ def _com_google_protobuf():
 def _com_github_grpc_grpc():
     _repository_impl("com_github_grpc_grpc")
 
+    if "com_github_nanopb_nanopb" not in native.existing_rules():
+        native.new_http_archive(
+            name = "com_github_nanopb_nanopb",
+            build_file = "@com_github_grpc_grpc//third_party:nanopb.BUILD",
+            strip_prefix = "nanopb-f8ac463766281625ad710900479130c7fcb4d63b",
```

Comment:
should we move SHA into repository_locations.bzl?

Reply:
Yeah, ideally we do this.